demos/Demo Box.hpp

```diff
@@ -0,0 +1,13 @@
+//
```

Comment:
Why this file is here?
It implements nothing and has completely different notations than OpenRT has: we do not use .hpp files, only .h, we use `#pragma once` instead of `#ifndef`. I've got the impression that this file is a copy-paste from somewhere.

Reply:
This file I created for a new demo(before you told me about the idea of having all Profcedural textures in one file). My Xcode creates has a template that he does automatically **#ifndef** format. Also my Xcode creates .hpp files instead of .h files.
Then I changed them manually to .h
I am going to delete this file, it didn't appear in my pull request, sorry for that

Reply:
Ah, OK. I created so far all the files manually, and did not know that XCode generates hpp with ifdefs automatically :-)

Reply:
I understand, sorry for the confusion
 please Professor whenever you have time could you please tell me about how I should think about the the conservation of texture problem, I will let you know in slack what are the approaches that I have made
 Thank you so much, I really know that you don't have time